config/crd/bases/flows.netobserv.io_flowcollectors.yaml

```diff
@@ -1432,24 +1432,35 @@ spec:
                   "agent" property is set to "ipfix".
                 properties:
                   cacheActiveTimeout:
-                    default: 60s
+                    default: 20s
                     description: CacheActiveTimeout is the max period during which
                       the reporter will aggregate flows before sending
                     pattern: ^\d+(ns|ms|s|m)?$
                     type: string
                   cacheMaxFlows:
-                    default: 100
+                    default: 400
                     description: CacheMaxFlows is the max number of flows in an aggregate;
                       when reached, the reporter sends the flows
                     format: int32
                     minimum: 0
                     type: integer
+                  forceSampleAll:
+                    default: false
+                    description: It is not recommended to sample all the traffic with
+                      IPFIX, as it may generate cluster instability. If you REALLY
+                      want to do that, set this flag to true. Use at your own risks.
+                      When it is set to true, the value of "sampling" is ignored.
+                    type: boolean
                   sampling:
                     default: 400
                     description: Sampling is the sampling rate on the reporter. 100
-                      means one flow on 100 is sent. 0 means disabled.
+                      means one flow on 100 is sent. To ensure cluster stability,
+                      it is not possible to set a value below 2. If you really want
+                      to sample every packet, which may impact the cluster stability,
+                      refer to "forceSampleAll". Alternatively, you can use the eBPF
+                      Agent instead of IPFIX.
                     format: int32
-                    minimum: 0
+                    minimum: 2
```

Comment:
There are no issues on `eBPF` side so I would not force this minimum here.

Moreover, I would prefer a realistic minimum based on perf tests for each agent more than a workaround here. Thoughts ?

Reply:
Sampling for ebpf and ipfix are two different configs, so no problem on that side.

I don't think we want to introduce minimums in general, ideally, users should be able to set whatever they want and be careful about what they do. But the issue we found is critical enough (like, put a cluster down) and happened quite consistently with ipfix' sampling=1, so that's why we take extra measures to prevent it to happen in that specific situation. The stability really isn't the same between sampling=1 and sampling=2.

Reply:
Oh sorry I though it was a common field. I agree the user should understand what he does and not able to put the cluster down.

What about adding some numbered recommendations in the descriptions based on performance results as a followup task ?

Reply:
yes, that's something we need to document anyway. Captured here: https://issues.redhat.com/browse/NETOBSERV-493